File: fixtures/conformance/csharp/deep_nesting.cs
public class deep_nesting {
    public static int deep_nesting(int[] items) {
        for (int a = 0; a < items.Length; a++) {
            if (a > 0) {
                while (a > 0) {
                    try {
                        if (a > 0) {
                            return a;
                        }
                    } finally {
                    }
                }
            }
        }
        return 0;
    }
}

File: fixtures/conformance/swift/deep_nesting.swift
func deep_nesting(x: Int) -> Int {
    for _ in 0..<x {
        if x > 0 {
            while x > 0 {
                do {
                    if x > 0 {
                        return x
                    }
                }
            }
        }
    }
    return x
}
